### ISSUE TITLE
Feature request: Notifications for receiving files

### ISSUE THREAD

**Issue by trymeouteh** [localsend/localsend#278]
### Is your feature request related to a problem? Please describe.

Notifications when receiving files

### Describe the solution you'd like

To get a system notification when another device requests sending files to you or if QuickSend is enabled, a notification to inform you that a file transfer has began.

System notification when files are received

System notification when files are sent to receiver

**Comment by zacharychin233** [comment]
Agree with that.

**Comment by saltedlolly** [comment]
Yes, please add this. It would be really useful, particularly on MacOS and Windows, where the app is running, but hidden, and Quick Save is On. Currently, if someone sends you a file without telling you, you have no way of knowing that they sent it.
